cgs_vmc/layers.py

```diff
@@ -405,3 +406,38 @@ def _build(self, inputs: tf.Tensor) -> tf.Tensor:
 
     batch_components = [self._mps_var[:, :, index] for index in index_inputs]
     return tf.stack(batch_components, axis=0)
+
+
+class GraphConvLayer(snt.AbstractModule):
+  """GraphConvLayer module with adjacency list"""
+
+  def __init__(
+      self,
+      output_channels: int,
+      adj: np.ndarray,
+      name: str = 'gnn_layer'):
```

Comment:
nit: As suggested by Sonnet library, default names of snt.AbstractModule should be  equal to class name converted from CamelCase to underscore case.  i.e. in this case it would be `graph_conv_layer `.